src/riskmatrix/models/risk_assessment.py

```diff
@@ -146,3 +146,7 @@ def __acl__(self) -> list['ACL']:
         return [
             (Allow, f'org_{self.risk.organization_id}', ['view']),
         ]
+
+
+class RiskMatrixAssessment(RiskAssessment):
+    nr: Mapped[int | None] = mapped_column(default=None)
```

Comment:
I would add this column to the base class. Proper Polymorphism in SQLAlchemy requires a column to store the polymorphic identity, so SQLAlchemy can load the correct version of the model when querying, but I don't think we gain anything from making this a separate class.